app/views/structurebuildingallowance/SbaCheckYourAnswersView.scala.html

```diff
@@ -32,6 +32,6 @@ <h1 class="govuk-heading-l">@messages("checkYourAnswers.heading")</h1>
 
     @govukButton(
         ButtonViewModel(messages("site.saveAndContinue"))
-        .asLink(routes.SbaClaimsController.onPageLoad(taxYear).url).withId("continue")
+        .asLink(routes.SbaClaimsController.onSubmit(taxYear).url).withId("continue")
```

Comment:
I don't think this is right. it should go to the SbaController, where it load and then once you make your selection on weather to add or not a new Sba claim, then on that form it should evaluate the boolean "value" and accordingly send the audit or not.